roles/contiv_network/tasks/v2plugin.yml

```diff
@@ -17,6 +17,15 @@
     /usr/bin/docker plugin install --grant-all-permissions {{contiv_v2plugin_image}} ctrl_ip={{node_addr}} control_url={{node_addr}}:{{netmaster_port}} vxlan_port={{vxlan_port}} iflist={{netplugin_if}} plugin_name={{contiv_v2plugin_image}} cluster_store={{cluster_store}} plugin_role={{run_as}}
   when: v2plugin_installed|failed
 
+- name: copy v2plugin.sh file
+  copy: src=v2plugin.sh dest=/usr/bin/v2plugin.sh mode=u=rwx,g=rx,o=rx
+
+- name: copy systemd units for v2plugin
+  copy: src=v2plugin.service dest=/etc/systemd/system/v2plugin.service
+
+- name: start v2plugin
+  systemd: name=v2plugin  enabled=yes
```

Comment:
Does this automatically do a `daemon-reload`?

Reply:
Above changes will make sure v2plugin.service and v2plugin.sh get install during  contiv-v2plugin installation. iv2plugin.service has dependency on etcd.service and docker service and get automatically restart on dependency service reload.